apps/studio/pages/project/[ref]/logs/postgrest-logs.tsx

```diff
@@ -15,6 +16,12 @@ export const LogPage: NextPageWithLayout = () => {
       queryType="postgrest"
       projectRef={ref as string}
       tableName={LogsTableName.POSTGREST}
+      EmptyState={
+        <LogsTableEmptyState
+          title="No results found"
+          description="By default only errors are logged into PostgREST logs. If you want to see HTTP requests, check the API Gateway logs."
```

Comment:
"...are captured in PostgREST logs..." ?

Reply:
^ i think thats better yup!